tutor/templates/kustomization.yml

```diff
@@ -59,4 +59,8 @@ configMapGenerator:
         app.kubernetes.io/name: redis
 {{ patch("kustomization-configmapgenerator") }}
 
+patchesStrategicMerge:
+- k8s/override.yml
+{{ patch("kustomization-patches-strategic-merge") }}
```

Comment:
Please document this patch in docs/reference/patches.rst.

Also, it would be great if you could include a section in the k8s docs on how to use this patch. Maybe give an example where you describe how to override PVC size?

Reply:
Will do